cassdegrees/ui/urls.py

```diff
@@ -15,9 +15,10 @@
 """
 from django.contrib import admin
 from django.urls import path
-from .views import index, sampleform
+from .views import index, sampleform, planList
 
 urlpatterns = [
     path('', index),
     path('sampleform/', sampleform),
+    path('list', planList),
```

Comment:
for consistency, should this end with `/`?

Reply:
Yes it should, +1 here